File: src/main/java/models/Parking.java
package models;

public class Parking {
    private int idParking;
    private String nomParking;
    private String localisationParking;
    private int place;


    public Parking() {
    }


    public Parking(int idParking, String nomParking, String localisationParking, int place) {
        this.idParking = idParking;
        this.nomParking = nomParking;
        this.localisationParking = localisationParking;
        this.place = place;
    }
    public Parking( String nomParking, String localisationParking, int place) {
        this.idParking = idParking;
        this.nomParking = nomParking;
        this.localisationParking = localisationParking;
        this.place = place;
    }

    public int getIdParking() {
        return idParking;
    }

    public void setIdParking(int idParking) {
        this.idParking = idParking;
    }

    public String getNomParking() {
        return nomParking;
    }

    public void setNomParking(String nomParking) {
        this.nomParking = nomParking;
    }

    public String getLocalisationParking() {
        return localisationParking;
    }

    public void setLocalisationParking(String localisationParking) {
        this.localisationParking = localisationParking;
    }

    public int getPlace() {
        return place;
    }

    public void setPlace(int place) {
        this.place = place;
    }

    // Méthode toString
    @Override
    public String toString() {
        return "Parking{" +
                "idParking=" + idParking +
                ", nomParking='" + nomParking + '\'' +
                ", localisationParking='" + localisationParking + '\'' +
                ", place=" + place +
                '}';
    }
}

File: Downloads/TuniGoPidevProject-GestionMoyTransport/src/Entity/Parking.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;

use App\Repository\ParkingRepository;

#[ORM\Entity(repositoryClass: ParkingRepository::class)]
#[ORM\Table(name: 'parking')]
class Parking
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(type: 'integer',name: 'idParking')]
    private ?int $idParking = null;

    public function getIdParking(): ?int
    {
        return $this->idParking;
    }

    public function setIdParking(int $idParking): self
    {
        $this->idParking = $idParking;
        return $this;
    }

    #[ORM\Column(type: 'string',name: 'nomParking', nullable: true)]
    private ?string $nomParking = null;

    public function getNomParking(): ?string
    {
        return $this->nomParking;
    }

    public function setNomParking(?string $nomParking): self
    {
        $this->nomParking = $nomParking;
        return $this;
    }

    #[ORM\Column(type: 'string',name: 'localisationParking', nullable: true)]
    private ?string $localisationParking = null;

    public function getLocalisationParking(): ?string
    {
        return $this->localisationParking;
    }

    public function setLocalisationParking(?string $localisationParking): self
    {
        $this->localisationParking = $localisationParking;
        return $this;
    }

    #[ORM\Column(type: 'integer',name: 'place', nullable: true)]
    private ?int $place = null;

    public function getPlace(): ?int
    {
        return $this->place;
    }

    public function setPlace(?int $place): self
    {
        $this->place = $place;
        return $this;
    }

}
